multitenancy/src/test/java/org/axonframework/extensions/multitenancy/configuration/MultiTenantEventProcessingModuleTest.java

```diff
@@ -210,6 +207,39 @@ void trackingEventProcessor() {
                                 .isPresent());
     }
 
+    @Test
+    void trackingEventProcessorNonMultiTenant() {
+        //noinspection unchecked
+        StreamableMessageSource<TrackedEventMessage<?>> mockedSource = mock(StreamableMessageSource.class);
+        TenantProvider tenantProvider = mock(TenantProvider.class);
+        MultiTenantStreamableMessageSourceProvider multiTenantStreamableMessageSourceProvider =
+                (source, processorName, tenantDescriptor, configuration) -> source;
+        configurer.registerModule(
+                new MultiTenantEventProcessingModule(tenantProvider, multiTenantStreamableMessageSourceProvider, null, (name) -> false)
```

Comment:
And for the `(name) -> false`, we could have a `MultiTenantEventProcessorPredicate.disableMultiTenancy()`, for example.

Reply:
Especially since it's key for this test that this lambda is set, having it as a named function clarifies the test a bit.